server.js

```diff
@@ -0,0 +1,109 @@
+import http from 'http';
+import f from './dist/events.json' assert { type: 'json' };
+
+const PORT = 8080;
+let eventsTodayList = [];
+const TODAY = new Date();
+
+function eventsToday(f) {
+    eventsTodayList = [];
+
+    f.events.forEach((event) => {
+        let flag = 0;
+        event.dates.forEach((date) => {
+            if (new Date(date.start) <= TODAY && new Date(date.end) >= TODAY) {
+                flag = 1;
+            }
+        });
+        if (flag == 1) {
+            // The Event Occurs Today
+            eventsTodayList.push({
+                name: event.summary,
+                desc: event.description,
+            });
+        }
+    });
+}
+
+const server = http.createServer((req, res) => {
```

Comment:
The intention for the index was to have an index.html, and not a server generated file. Sorry if that wasn't clear but because it's hosted on GitHub we can't use node. It has to be a built index.html file. One can be used for dev purposes though.

Reply:
Oh alright. Do you suggest I keep this sevrer.js file or transfer all the logic to the index.html file itself?

Reply:
Considering GitHub hosts the file I think keeping a server locally to host it is a great idea, but the logic should live in index.html to determine the current event, etc.